doc/languages-frameworks/gradle.section.md

```diff
@@ -0,0 +1,140 @@
+# Gradle {#gradle}
+
+Gradle is a popular build tool for Java/Kotlin. Unfortunately, despites
```

Comment:
```suggestion
Gradle is a popular build tool for Java/Kotlin. Unfortunately, despite
```

Will apply later, don't want to push too much, it will spam Github with 31 commits

Reply:
You could also just push those as regular commits and simply squash them later

Reply:
you're right